cumulusci/cumulusci.yml

```diff
@@ -233,6 +233,12 @@ tasks:
         options:
             suites: tests
         group: Robot Framework
+    robot_documentation:
+        description: Generates documentation for project keyword files
+        class_path: cumulusci.tasks.robotframework.RobotDoc
+        options:
+            outputdir: docs
+        group: Robot Framework
     robot_testdoc:
         description: 
```

Comment:
It seems potentially confusing to provide 2 different tasks for generating robot framework documentation. Is there any reason we should keep the old one? If not, I would be inclined to completely replace it with the new one, but keep the same task name (robot_testdoc) and option names (path and output) to smooth the transition.

Reply:
I agree. The only real problem with keeping the same arguments is that my script takes the path to more than one file. That's a pretty minor thing, though.

Reply:
It's backwards compatible as long as no one was using commas in their path, which seems unlikely.

Reply:
In the code I pushed earlier today I completely replaced the old robot_libdoc task with the new code. The new task takes the same arguments as the old task.